esterni/manuale-utente/components/mobile-application.tex

```diff
@@ -0,0 +1,134 @@
+\end{figure}
+
+Se l'utente non si è mai registrato in Stalker, questo form è l'unico modo per farlo!\\
+L'utente per registrarsi deve inserire delle semplici informazioni personali:
+
+\begin{itemize}
+    \item inserire il nome.
+    \item inserire il cognome.
+    \item inserire la data di nascita.
+    \item inserire l'email, non precedentemente utilizzata in Stalker.
+    \item inserire la password, che deve essere alfanumerica con almeno 8 caratteri e al massimo 32.
+    \item premere il pulsante sign up per effettuare la registrazione.
+\end{itemize}
+
+Si avvisa inoltre che non è possibile registrarsi più volte con la stessa email.\\
+Se questi vincoli non saranno rispettati l'utente non potrà registrarsi.\\
+Se l'utente possiede delle credenziali, ha la possibilità di accedere, grazie all'apposito form disponibile dopo aver cliccato su \textit{Already have an account? Login}.
+\newpage
+
+\subsubsection{Pagina iniziale}%
+\label{sub:pagina_iniziale}
+
+% \begin{figure}[H]
+%     \centering
+%     \includegraphics{img/mobile-app/pagina-iniziale.png}
+%     \caption{Pagina iniziale}%
+%     \label{fig:mobile_app_pagina_iniziale}
+% \end{figure}
+
+Lorem ipsum
+
+% add other functionalities 
+\newpage
+\subsection{Risoluzione dei problemi}
+
+\subsubsection{Accesso all'applicazione non disponibile}
+
+Nel caso ci fossero problemi con il collegamento all'app, vi consigliamo di controllare se al momento è attiva una rete internet nel vostro smartphone Android.
+Nel caso in cui la rete sia abilitata vi preghiamo di riprovare l’accesso in un secondo momento.
+Se il problema dovesse persistere vi preghiamo di segnalarcelo nelle modalità discusse nella sezione \emph{Supporto tecnico}.
+
+\subsubsection{Errore nel reperimento della posizione}
+Nel caso il vostro smartphone non riuscisse a reperire la vostra posizione, vi consigliamo di verificare se al momento è attiva la geolocalizzazione nel vostro smartphone.
+Se la geolocalizzazione dovesse essere attiva, vi consigliamo di provare a seguire i seguenti passaggi:
+\begin{itemize}
+    \item Accedere alle impostazioni del vostro smartphone, tramite l'apposita icona oppure tramite la tendina delle notifiche.
+    \item Accedere alla pagina \emph{applicazioni} oppure \emph{app e notifiche}. Nel caso non troviate queste pagine dalle impostazioni, vi consigliamo di consultare
+        il manuale del vostro smartphone e ricercare la pagina rilevante le applicazioni installate.
+    \item Cercare tra le applicazioni in lista \emph{Stalker} e selezionarlo.
+    \item Accedere alla sezione \emph{autorizzazioni} o \emph{permessi} e verificare se i permessi riguardante la geolocalizzazione sono attivi.
+    \item Nel caso non fossero attivi, permettere la geolocalizzazione a \emph{Stalker}.
+\end{itemize}
+Nel caso nemmeno i passi indicati siano stati utili per il funzionamento di \emph{Stalker} verificare il corretto funzionamento della geolocalizzazione attraverso
+altre applicazioni. \\
```

Comment:
OT: Ragazzi se le righe che state scrivendo sono troppo lunghe e non ci stanno nell'editor non le si rende più leggibili andando a capo a caso in sto modo, usate il word wrap (su VS code basta premere ALT + Z)